rollup.config.js

```diff
@@ -47,5 +48,6 @@ export default {
       ENV: JSON.stringify(process.env.NODE_ENV || 'development'),
     }),
     (process.env.NODE_ENV === 'production' && uglify()),
+    (process.argv.indexOf('--live') !== -1 && livereload('build')),
```

Comment:
http://node.green/#Array-prototype-includes

You probably can, but it's pretty new. Tested it here and indeed, it's not supported on node v4.4.7